### DISCUSSION TITLE
A dedicated delivery package for NBCode

### DISCUSSION THREAD

**Discussion by mickaelistria** [apache/netbeans#6614] [General]
### Description

At the moment, it seems difficult to download and try the NetBeans NBCode Language Server/Debug Adapter. The only ways to do so are to either build nbcode locally, or to extract it from some VSCode Marketplace extension.

It would be convenient if the nbcode delivery (eg a zip) were made available for download, even just as a build artifact available from https://ci-builds.apache.org/job/Netbeans/job/netbeans-linux/lastSuccessfulBuild/artifact .

### Use case/motivation

I want to be able to try a recent version of nbcode in other IDEs (eg Eclipse IDE) more easily.

### Are you willing to submit a pull request?

Yes

**Comment by neilcsmith-net** [comment]
The nbcode delivery as in the vsix file?

That is available at https://downloads.apache.org/netbeans/netbeans-vscode-ext/19.0.301/

And it is normally available for release candidates at https://nightlies.apache.org/netbeans/candidate/vsix/ As mentioned in #6581 this isn't available for 20-rc1 due to a build error which is now fixed.  It should be up when 20-rc2 is announced later this week.

**Reply by mickaelistria** [reply]
Thanks for pointing to the latest build fo the vsix, that's already helpful as latest `nbcode` is embedded in here and can be extracted from it.
But iideally, I don't want the full vsix file but just the language server/debug adapter, `nbcode` NetBeans application folder, which is later embedded in the .vsix.
